docs/builtins.rst

```diff
@@ -0,0 +1,217 @@
+================
+Builtins in L20n
+================
+
+L20n ships with a number of built-in functions that can be used from within
+localization messages.
+
+The list of built-ins is extensible and environments may want to add additional
+built-ins designated to aid localizers writing translations targeted
+for a given environment.
+
+
+How to use Built-ins
+====================
+
+Localizers can reference builtin in a message within a placeable and either
+display the return value, or use it in the selector expression to select
+the right value.
+
+Examples::
+
+
+  today-is = Today is { DATETIME($date, weekday=long) } 
```

Comment:
Our current syntax requires `:` for the keyword arguments and quotes:

    today-is = Today is { DATETIME($date, weekday: "long") } 

https://github.com/l20n/l20n.js/blob/3ba0b9ad8f20a9540eb7ef6c4bda4604af4677e1/src/ftl/ast/parser.js#L503

We should probably change the EBNF grammar: https://github.com/l20n/spec/blob/126b422ae146cdb81eed38484eca8d732cec46a6/grammar.ebnf#L49